packages/map-popup/src/util.ts

```diff
@@ -63,6 +66,14 @@ export function makeDefaultGetEntityName(
         },
         { name: stationName }
       );
+    } else if (
+      stopIdAgencyMap &&
+      entity.id in stopIdAgencyMap &&
+      "code" in entity
+    ) {
+      stationName = `${stationName} (${stopIdAgencyMap[entity.id].name} ${
```

Comment:
Do we need to pass an entire map if the only important thing is the agency name of the stop? Can we just pass an agency id to the popup?